app/views/MonthlyPaymentAmount.scala.html

```diff
@@ -67,19 +67,19 @@
         ),
             title = Text(Messages.`There is a problem`.show)))
     }
-    @formWithCSRF(action = routes.MonthlyPaymentAmountController.monthlyPaymentAmountSubmit) {
-
-        <h1 class="govuk-heading-xl">@title</h1>
 
-        <p class="govuk-body">
-            @{Messages.MonthlyPaymentAmount.`The miminum payment you can make is ...`(minimumPayment).show}
-        </p>
+    <h1 class="govuk-heading-xl">@title</h1>
+    <p class="govuk-body">
+    @{Messages.MonthlyPaymentAmount.`The miminum payment you can make is ...`(minimumPayment).show}
+    </p>
+    @govukDetails(Details(
+        summary = Text(Messages.MonthlyPaymentAmount.`I can’t afford the minimum payment`.show),
+        content = HtmlContent(detailsContent)
+    ))
 
-        @govukDetails(Details(
-            summary = Text(Messages.MonthlyPaymentAmount.`I can’t afford the minimum payment`.show),
-            content = HtmlContent(detailsContent)
-        ))
+    @formWithCSRF(action = routes.MonthlyPaymentAmountController.monthlyPaymentAmountSubmit) {
 
+        <label for="@key">
```

Comment:
is there a way of using `label` in the parameters to `Input` here:
https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/input/Input.scala#L33
? Does it clash in any way with the label that's created already by the input here?
https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukInput.scala.html#L25

Reply:
If I use the label param in the input, it puts some extra text in which doesn't work, can we catch up after standup?